--- postgres-xl-9.5r1.4/src/backend/utils/misc/postgresql.conf.sample	2016-11-29 09:11:34.000000000 +0000
+++ postgres-xl-9.5r1.4.working/src/backend/utils/misc/postgresql.conf.sample	2016-12-22 02:35:04.671514471 +0000
@@ -324,18 +324,21 @@
 					# stderr, csvlog, syslog, and eventlog,
 					# depending on platform.  csvlog
 					# requires logging_collector to be on.
+log_destination = 'stderr'
 
 # This is used when logging to stderr:
 #logging_collector = off		# Enable capturing of stderr and csvlog
 					# into log files. Required to be on for
 					# csvlogs.
 					# (change requires restart)
+logging_collector = on
 
 # These are only used if logging_collector is on:
 #log_directory = 'pg_log'		# directory where log files are written,
 					# can be absolute or relative to PGDATA
 #log_filename = 'postgresql-%Y-%m-%d_%H%M%S.log'	# log file name pattern,
 					# can include strftime() escapes
+log_filename = 'pgxl-%Y-%m-%d_%H%M%S.log'
 #log_file_mode = 0600			# creation mode for log files,
 					# begin with 0 to use octal notation
 #log_truncate_on_rotation = off		# If on, an existing log file with the
@@ -346,11 +349,14 @@
 					# or size-driven rotation.  Default is
 					# off, meaning append to existing files
 					# in all cases.
+log_truncate_on_rotation = on
 #log_rotation_age = 1d			# Automatic rotation of logfiles will
 					# happen after that time.  0 disables.
+log_rotation_age = 1d
 #log_rotation_size = 10MB		# Automatic rotation of logfiles will
 					# happen after that much log output.
 					# 0 disables.
+log_rotation_size = 0
 
 # These are relevant when logging to syslog:
 #syslog_facility = 'LOCAL0'
